File: src/ROT13HiddenTest.cs
using NUnit.Framework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Rot13Challenge.src
{
    [TestFixture]
    public class ROT13HiddenTest : ROT13
    {
        [Test]
        public void EncryptTest()
        {
            Assert.AreEqual("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz", Encrypt(Encrypt("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz")));
            Assert.AreEqual("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz", Encrypt("NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm"));
        }
    }
}

File: Tests/ROT13HiddenTest.vb
Imports NUnit.Framework

<TestFixture()>
Public Class ROT13HiddenTest
    Inherits ROT13

    <Test()>
    Public Sub EncryptTest()
        Const LookupTable As String = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
        Const TransformTable As String = "NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm"

        Assert.AreEqual(TransformTable, Encrypt(LookupTable))
        Assert.AreEqual(LookupTable, Encrypt(TransformTable))
    End Sub
End Class
